Classes/Booking.cs

```diff
@@ -1,24 +1,31 @@
-﻿namespace AirportTicketBooking.Classes
+﻿using AirportTicketBooking.Controllers;
+
+namespace AirportTicketBooking.Classes
 {
     internal class Booking
     {
         internal int BookingId { get; set; }
-        internal Flight Flight { get; set; }
+        internal int FlightId { get; set; }
 
-        internal Passenger Passenger { get; set; }
+        internal int PassengerPassportId { get; set; }
 
         internal DateTime BookingDate { get; set; }
 
         internal TicketClassType.ClassType ClassType { get; set; }
```

Comment:
How you decided that the ClassType is part of the Booking entity rather than the Flight entity for example?

Reply:
Note: I see it's fine to include it in the booking entity. I just wanted to know what you thought about it.